src/schemas.ts

```diff
@@ -800,6 +800,48 @@ const foxglove_LaserScan: FoxgloveMessageSchema = {
   ],
 };
 
+const foxglove_EventKeyValue: FoxgloveMessageSchema = {
+  type: "message",
+  name: "EventKeyValue",
+  description: "a single key-value pair of metadata for an event",
+  fields: [
+    {
+      name: "key",
+      type: { type: "primitive", name: "string" },
+      description: "key for this metadata",
+    },
+    {
+      name: "value",
+      type: { type: "primitive", name: "string" },
+      description: "value of the metadata",
+    },
+  ],
+};
```

Comment:
Adding an identical KeyValuePair in https://github.com/foxglove/schemas/pull/17. Want to remove Event from the name and "metadata" from the descriptions so we can reuse it in both places?

Reply:
Seems ok for now. The Event schema might evolve more over time (e.g. to support other structured data like nested arrays / structs) but we can cross that bridge when we get to it.